addons/hr_contract/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,25 @@
+---Fields in module 'hr_contract'---
+hr_contract  / hr.contract              / active (boolean)              : NEW 
+# DONE: post-migration: filled as True by default
```

Comment:
No need. Its default value is already True, filled when adding the field in the regular update process.